src/components/Articles/CompanyProvisions.vue

```diff
@@ -0,0 +1,246 @@
+<template>
+  <div id="company-provisions">
+    <v-container class="pa-0 ma-0" v-if="!isEditing">
+      <v-row no-gutters>
+        <v-col class="pr-0 pl-0 pt-0" cols="3">
+          <label class="define-company-provisions-title">Pre-existing Company Provisions</label>
+          <v-chip v-if="hasProvisionsRemovedPropsChanged" x-small label color="primary" text-color="white">
+            CHANGED
+          </v-chip>
+        </v-col>
+        <v-col cols="7" class="pt-0 pl-0 info-text" v-if="provisionsRemoved">
+          The company has resolved that none of the Pre-existing Company Provisions are to apply to this company.
+        </v-col>
+        <v-col cols="7" class="pt-0 pl-0 info-text" v-else>
+          This company has Pre-existing Company Provisions.
+        </v-col>
+        <v-col mt-n2 cols="2" class="pt-0 align-right" v-if="!hasProvisionsRemovedPropsChanged">
+          <v-btn id="correct-company-provisions" text color="primary" @click="isEditing = true">
+            <v-icon small>mdi-pencil</v-icon>
+            <span>{{ editLabel }}</span>
+          </v-btn>
+        </v-col>
+        <v-col pt-0 mt-n2 cols="2" class="align-right" v-else>
```

Comment:
These spacing helper classes need to be in the class attribute.

Reply:
Also on line 56 below.

Reply:
And line 84.

Please check to see if you missed any others.

Reply:
👍